src/lib/model-matcher.ts

```
@@ -0,0 +1,139 @@
+  consola.debug(`Available models: ${availableModelIds.join(", ")}`)
+
+  // Try exact match first
+  if (availableModelIds.includes(requestedModel)) {
+    return requestedModel
+  }
+
+  // Normalize the requested model
+  // 1. Replace underscores with hyphens
+  // 2. Remove date suffix (8 digits at the end)
+  // 3. Replace version numbers: 4-5 -> 4.5
+  let normalizedRequested = requestedModel
```

ESLint

[error] 'normalizedRequested' is never reassigned. Use 'const' instead.

(prefer-const)